crates/ruff/src/registry.rs

```diff
@@ -792,6 +798,9 @@ pub enum Linter {
     /// [flake8-use-pathlib](https://pypi.org/project/flake8-use-pathlib/)
     #[prefix = "PTH"]
     Flake8UsePathlib,
+    /// [flake8-todos](https://github.com/orsinium-labs/flake8-todos/)
+    #[prefix = "T"]
```

Comment:
I think it's better to avoid single letter prefix for posterity, maybe `TOD`, `TDO`, `TD`?

Reply:
Do we use our own error codes? If so, I'm happy to change it - I only used `T` because[flake8_todos](https://github.com/orsinium-labs/flake8-todos) uses it.

Reply:
> Do we use our own error codes?

We do to avoid collisions as there are multiple error groups in `ruff` as compared to a single group in individual plugins. Example: [flake8-tidy-imports (TID)](https://beta.ruff.rs/docs/rules/#flake8-tidy-imports-tid) vs [Plugin repo (I)](https://github.com/adamchainz/flake8-tidy-imports#rules). But, let @charliermarsh's take the final call on this.

Reply:
Yeah that's right -- we typically use three-letter prefixes to avoid collisions, even if it deviates from upstream. I guess `TOD` or `TDO` are equally reasonable here.

Reply:
Done